src/Agent.php

```diff
@@ -714,8 +714,15 @@ private static function cleanAgent(Agent $agent, array $config): bool
          * @phpstan-var array{label: string, item_action: boolean, render_callback: callable, action_callback: callable}[] $actions
          */
         foreach ($plugin_actions as $plugin => $actions) {
-            if (is_array($actions) && \Plugin::isPluginActive($plugin)) {
+            if (is_array($actions) && Plugin::isPluginActive($plugin)) {
                 foreach ($actions as $action) {
+                    if (!is_callable($action['action_callback'] ?? null)) {
+                        trigger_error(
+                            sprintf('Invalid plugin "%s" action callback for "%s" hook.', $plugin, Hooks::STALE_AGENT_CONFIG),
+                            E_USER_WARNING
+                        );
+                        continue;
+                    }
                     if ($action['item_action']) {
                         $need_item = true;
                     }
```

Comment:
We could probably consider that a plugin that acts on stale agents will require the corresponding item in most cases. Having a `item_action` param would require to write a documentation about it, and developpers does not always read documentation. The fact that plugins may receive a different payload depending on `stale_agents_status` would probably be incorrectly handled for developpers that does not understand the usage of `item_action`. So, IMHO, removing it may prevent issues on plugins.

```suggestion
```